daemon/src/connection.rs

```diff
@@ -172,3 +175,49 @@ impl Actor {
 }
 
 impl xtra::Actor for Actor {}
+
+pub async fn connect(
+    mut maker_online_status_feed_receiver: watch::Receiver<ConnectionStatus>,
+    connection_actor_addr: xtra::Address<Actor>,
+    maker_identity_pk: x25519_dalek::PublicKey,
+    maker_addresses: Vec<SocketAddr>,
+) {
+    loop {
+        if maker_online_status_feed_receiver.borrow().clone() == ConnectionStatus::Offline {
+            tracing::info!("No connection to the maker, attempting to connect:");
+            'reconnect: loop {
```

Comment:
Is trying to connect to all resolved addresses really a reconnect?

Reply:
renamed to `connect`